demos/browser/app/meetingV2/styleV2.scss

```diff
@@ -425,6 +425,22 @@ svg {
   }
 }
 
+.video-preview {
+  width:auto;
+  height:82px;
+  margin-left:35%;
+  margin-top: 7px;
+}
+
+@media (min-width: 576px){
+  .video-preview {
+    width:100%;
+    height:82px;
+    margin-left:0%;
```

Comment:
Recommend using the same unit for zero, or you can just use `0` without any unit.
```
margin-left: 0;
margin-top: 0;
```